src/vs/platform/actions/browser/menuItemActionItem.ts

```diff
@@ -45,6 +55,10 @@ class AltKeyEmitter extends Emitter<boolean> {
 		this.fire(this._isPressed);
 	}
 
+	suppressAltKeyUp() {
+		this._suppressAltKeyUp = true;
+	}
+
 	@memoize
 	static getInstance(contextMenuService: IContextMenuService) {
```

Comment:
@isidorn unrelated, related but using `@memoize` is risky because I am pretty sure it memoizes without looking at the arguments and I am not sure there is only one `contextMenuService` around (i might be mistaken...)

Reply:
@jrieken yes it ignores the arguments. Thanks for letting me know, will try to fix.